app/assets/javascripts/listings/directives/lottery-preference.html.slim

```diff
@@ -0,0 +1,17 @@
+.content-card
+  header.content-card_ordered
+    span.content-card_number
+      ' {{preference.order}}
```

Comment:
How were you able to map preference without using attrs? Was that passed in naturally from ng-repeat= preference in preferences?